app/assets/javascripts/govuk_publishing_components/analytics/mailto-link-tracker.js

```diff
@@ -17,6 +17,11 @@
       }
 
       var href = element.getAttribute('href')
+
+      if (!href) {
```

Comment:
Do we also want to add a test in `mailto-link-tracker.spec.js` with an anchor without an href, to make sure we check this conditional statement is preserved over time.

Reply:
Good suggestion, have modified the initial test to check this.